harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
I don't think this is worthwhile. test262 tests are meant to be run in isolation, and I appreciate that the verify helpers make this very clear. Relying on hosts to implement all the semantics necessary for restoring a property descriptor is not sound testing practice; it makes a lot of assumptions about correct implementation which are intimately related to the very assumptions we are trying to test here.

I think this PR is better off closed, or converted into a PR with comments. I'd also support not restoring the originals in the other verify helpers, to improve testing hygiene in those.